packages/core/elasticsearch/core-elasticsearch-server-internal/src/elasticsearch_config.ts

```diff
@@ -94,7 +94,7 @@ export const configSchema = schema.object({
       }),
     ],
     {
-      defaultValue: ['authorization'],
+      defaultValue: ['authorization', 'es-client-authentication'],
```

Comment:
__note:__ the time has come :slightly_smiling_face: See https://www.elastic.co/guide/en/elasticsearch/reference/8.11/jwt-auth-realm.html#jwt-realm-configuration